fortinet_fortigate/icon_fortinet_fortigate/actions/add_address_object_to_address_group/action.py

```diff
@@ -30,11 +30,25 @@ def run(self, params={}):
         try:
             response.raise_for_status()
         except Exception as e:
-            raise PluginException(cause=f"Add address object to address group failed: {endpoint}\n",
-                                  assistance=response.text,
-                                  data=e)
+            json_obj = {}
```

Comment:
No need to declare this - did you get a warning from your editor? I'm guessing it was because it can't tell that line 37 is terminal.